src/charm.py

```diff
@@ -947,14 +950,6 @@ def _on_start(self, event: StartEvent) -> None:
 
         self.unit.set_workload_version(self._patroni.get_postgresql_version())
 
-        try:
-            # Set up the postgresql_exporter options.
-            self._setup_exporter()
-        except snap.SnapError:
-            logger.error("failed to set up postgresql_exporter options")
-            self.unit.status = BlockedStatus("failed to set up postgresql_exporter options")
```

Comment:
There will no longer be a blocked status over this, but the call in `_on_peer_relation_changed()` will retry if not successful.